cypress/Shared/MeasureGroupPage.ts

```diff
@@ -161,18 +161,21 @@ export class MeasureGroupPage {
     //Risk Adjustment variables
     public static readonly riskAdjustmentDefinitionSelect = '[data-testid=ArrowDropDownIcon]'
     public static readonly riskAdjustmentDefinitionDropdown = '[id="risk-adjustment-listbox"]'
+    public static readonly riskAdjustmentTextBox = '.MuiInputBase-root > #risk-adjustment'
     public static readonly cancelIcon = '[data-testid=CancelIcon]'
     public static readonly riskAdjustmentDescriptionTextBox = '[class=risk-description]'
+    public static readonly saveRiskAdjustments = '[data-testid="measure-Risk Adjustment-save"]'
+    public static readonly riskAdjustmentSaveSuccessMsg = '[class="toast success"]'
 
     //Supplemental data elements
     public static readonly supplementalDataDefinitionSelect = '[data-testid=supplementalDataElements]'
     public static readonly supplementalDataDefinitionDropdown = '#supplementalDataElements-listbox'
     public static readonly supplementalDataDefinitionDescriptionTextBox = '[class="sc-iqcoie glofVe"]'
     public static readonly saveSupplementalDataElements = '[data-testid="supplementalDataElement-save"]'
     public static readonly supplementalDataElementsSaveSuccessMsg = '[data-testid="supplementalDataElement-success"]'
-    public static readonly supplementalDataElementsDiscardChangesBtn = '[data-testid=cancel-button]'
-    public static readonly supplementalDataElementsDiscardChangesConfirmationMsg = '.MuiDialogContent-root'
-    public static readonly supplementalDataElementsDiscardChangesContinueBtn = '[data-testid="discard-dialog-continue-button"]'
+    public static readonly discardChangesBtn = '[data-testid=cancel-button]'
```

Comment:
don't we have global variables for these elements now?

Reply:
these variables are different than the ones on Global page